pkg/build/util/util.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+import (
+	"fmt"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+)
+
+// GetBuildPodName returns name of the build pod.
+func GetBuildPodName(build *buildapi.Build) string {
+	return fmt.Sprintf("pod-%s", build.Name)
```

Comment:
btw. does it need to have "pod-" prefix? how is that usefull? when I do osc get pods I know that all things returned are pods ;-)